prover/Cargo.lock

```diff
@@ -7227,7 +7266,7 @@ dependencies = [
 [[package]]
 name = "zksync_protobuf_build"
 version = "0.1.0"
-source = "git+https://github.com/matter-labs/era-consensus.git?rev=da015d4c94b19962bc11622b6cc256e214256555#da015d4c94b19962bc11622b6cc256e214256555"
+source = "git+https://github.com/matter-labs/era-consensus.git?rev=49b1a98f80d0e9f74fdceadece4283e745c71599#49b1a98f80d0e9f74fdceadece4283e745c71599"
```

Comment:
pass-by comment: did you change file on purpose?

Reply:
Somewhat. I have both the main and prover workspaces set up in the IDE, so the prover lockfile is kept up to date. We don't have a check that it's up to date in CI, and it often gets outdated in practice.